ansible/roles/node/tasks/localBuildInstall.yml

```diff
@@ -1,7 +1,7 @@
 ---
 - name: Copy node binaries
   copy:
-    src: "{{ localBuildOutput }}/{{ item }}"
+    src: "{{ localBuildOutput }}/{{ item }}" #docker builds, use: kubernetes/_output/dockerized/bin/linux/amd64/{{ item }}
```

Comment:
I don't like this comment. It suggests that someone might want to make a change here. Which would be a bad idea, given we use localBuildOutput in more than this one location....